OxfordBodleian/Juel-Jensen/BDLaethd23.xml

```diff
@@ -154,21 +260,201 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+                                    <desc type="RecordReconciliation">Amharic reconciliation record mentioning many names.</desc>
+                                    <locus target="#19va"/>
+                                    <q xml:lang="am">የብላታ፡ ክንፋ፡ ምድር፡ ከኖረኀ፡ <add place="above">ምክሩ፡</add>
+                                    የታረቍበት፡ ዕፃ፡ ተልቅናውን፡ መሬት፡ እኩል፡ መሬት፡ ታረቁ፡ በከብተዎ፡ አለዳ፡ 
+                                    አለዕንግዳ፡ <add place="above">ሲበሉ፡</add> አስታራቂዎች፡ ፭ቱ፡ ሸማግሎች፡ 
+                                    ነጮ፡ ዕንቈ፡ ሥላሴ፡ ወልደ፡ ዮሐንስ፡ በውቀቱ፡ ጕበና፡ ኃይሉ፡ ብላታ፡ ለትጹን፡ ብላታ፡ 
+                                    ገጹ፡ <sic resp="JK">ናቼው፡</sic> በረምኃ፡ ጊዮርጊስ፡ ግብዝና፡ በደብረ፡ ሲና፡ 
+                                        ገበዝ፡ <add place="above">በ</add>ባይጠፋኝ፡ በ፬ቱ፡ ዕንስሳ፡ በእብሩ፡ 
+                                    ይግዛው፡ ግብዝና፡ የአቦና፡ የጊዮርጊስ፡ ገበዝ፡ ቄስ፡ ወልደ፡ ዕ፳ኤል፡ በዕነዚኅ፡ 
+                                    ሁሉ፡ ፊት፡ ተጻፈ፡ በዘሚካኤል፡ ጭቅነት፡ በወልደ፡ አብ፡ በአስራት፡ ጭቅነት፡ 
+                                        <add place="above">በ</add>ማትያስ፡ ገብረ፡ ማርያም፡ በሊቀላዎስ፡ 
+                                        ጭቅነት፡ በ<unclear>ን</unclear>ዋይ፡ ሐፃኑ፡ የለባሲት፡ ገንታ፡ 
+                                        ማሞ፡ ገብሩ፡ ዋሱ፡ ለኖረኀ<supplied reason="omitted">፡</supplied>
+                                        ፈንታ፡ ወለተ፡ ሥላሴ፡ ነው፡ ጸሐፊው፡ ደብተራ፡ ገብረ፡ ክርስቶስ፡ 
+                                        <gap reason="illegible" unit="chars" quantity="5"/>
+                                        ብላታ፡ <unclear>ደርሳኀ፡</unclear> ነው፡ 
+                                        <gap reason="illegible" unit="chars" quantity="6"/>ስ፡ 
+                                        ዕንዶይፋ<gap reason="illegible" unit="chars" quantity="1"/>
+                                    </q>
+                                </item>
+                                
+                                <item xml:id="a4">
+                                    <desc type="MixedNote">Amharic note written in a careful hand mentioning that the manuscript was written for
+                                        <persName ref="PRS14785EhetaMaryam">ʾƎḫəta Māryām</persName>, the sister of 
+                                        King <persName ref="PRS5617IyasuII">ʾIyāsu II</persName>, during his reign, while she was
+                                        engaged in the construction of the church of <placeName ref="INS1023BadawoAragawi">Badāwo ʾAbuna ʾAragāwi</placeName>.
+                                        <!--AMHARIC TO BE DISCUSSED WITH SOMEONE WHO KNOWS IT BETTER-->
+                                    </desc>
+                                    <locus target="#26r"/>
+                                    <q xml:lang="am"><foreign xml:lang="gez">ዝንቱ፡ ደብረ፡ አረጋዊ፡ ዘተተክለ፡</foreign> 
+                                        <persName ref="PRS5617IyasuII"><roleName type="title">ባጼ፡</roleName> ኢያሱ።</persName> 
+                                        <persName ref="PRS2682BerhanM"><roleName type="title">እቴዬ፡</roleName> ምንትዋብ፡</persName> 
```

Comment:
Probably እቴጌ፡ ?

Reply:
No, it's quite clearly እቴዬ፡, actually. I also find this title in a reasonable number of Google hits, but I don't know if they are all errors or not… Should I sic it or can I leave it like this?